indexer/indexer.toml

```diff
@@ -10,11 +10,15 @@ l1-rpc = "${INDEXER_RPC_URL_L1}"
 l2-rpc = "${INDEXER_RPC_URL_L2}"
 
 [db]
-host = "postgres"
+host = "$INDEXER_DB_HOST"
+# this port may be problematic once we depoly
```

Comment:
If i had to guess, using the `postgresql://` scheme probably defaults to the 5432 ports

Quick google search confirms the default: https://www.prisma.io/dataguide/postgresql/short-guides/connection-uris -- we can remove this comment

Reply:
I think I've actually had an issue in the past if I remember correctly when deploying gateway-backend. Gateway-backend didn't work the first time I deployed it and I had to remove the port

Reply:
Definitely a chance I'm remembering wrong though it was a long time ago